vrk-kpa/suomidigi
# SUOPA-137 Mobile menu

To test: 
- `make fresh`
- Log in to the site and clear the caches from the admin UI
- Go to mobile device mode and test that the mobile menu works as in proto 
   - The gray area in proto under the close menu button should be gray with some opacity. so it's a feature not a bug. :)
